guides/release/components/block-content.md

```diff
@@ -22,6 +22,24 @@ This is equivalent to explicitly naming the default block using the named block
 </ExampleComponent>
 ```
 
+Or consider the following modal component which utilizes a content block and a footer block.
+
+```handlebars
+<div class="modal" role="dialog">
+    <button type="button">Close modal</button>
+    <div class="modal__content">
+        <:content>
+            This block will be replaced with the `{{yield to="content"}}` `<Modal />` content.
```

Comment:
This code example doesn't quite work?

Either the `<:content>` blocks are used as a direct child element of a component. Or this is defining the modal component itself, in which case it should just use `{{yield to="content"}}`.

I would suggest creating a code example of both the component definition as well as the usage.